client/js/file-upload/gdrive-picker.js

```diff
@@ -0,0 +1,87 @@
+const apiKey = 'AIzaSyC3wEaP6u8f6yKjraSz_V0rnVYKqc5BvG4';
```

Comment:
nit: add a comment like `// Since the Drive API is free to use, we don't need to hide our API key.`

Reply:
good catch. I actually wasn't sure why the API key didn't need to be hidden, but I couldn't figure out a way to hide it anyway